dask_image/ndfilters/_conv.py

```diff
@@ -13,17 +13,17 @@
 
 
 @_utils._update_wrapper(scipy.ndimage.filters.convolve)
-def convolve(image,
-             weights,
-             mode='reflect',
-             cval=0.0,
-             origin=0):
+def convolve(image, weights, mode="reflect", cval=0.0, origin=0):
     check_arraytypes_compatible(image, weights)
 
     origin = _utils._get_origin(weights.shape, origin)
     depth = _utils._get_depth(weights.shape, origin)
     depth, boundary = _utils._get_depth_boundary(image.ndim, depth, "none")
 
+    if mode == "wrap":
```

Comment:
```suggestion
    if mode == "wrap":  # Fixes https://github.com/dask/dask-image/issues/242
```

The reason is clear now, but future developers may wonder why this is a special case. Adding a note will save them time later.